openbb_platform/obbject_extensions/charting/openbb_charting/charting_router.py

```diff
@@ -1,145 +1,769 @@
 """Charting router."""
 
-import json
+# pylint: disable=too-many-arguments, too-many-locals, too-many-branches, too-many-statements, unused-argument, too-many-lines
```

Comment:
We should try to make our code comply with the linting as much as possible, instead of ignoring it.

Reply:
We could also set the rules of the linter to allow more arguments parameters and locals. "unused-argument" is basically a required import for every file.